state/pruner.go

```diff
@@ -116,8 +121,12 @@ func (p *Pruner) SetObserver(obs PrunerObserver) {
 
 func (p *Pruner) OnStart() error {
 	go p.pruneIndexesRoutine()
```

Comment:
```suggestion
```

Reply:
I can't suggest this, but basically this should be put in line 128 below (under `p.pruneABCIResponses()` ) , because it should be done only if the data companion is enabled.